next-i18next.config.js/index.js

```diff
@@ -1,20 +1,24 @@
+// The implementaion refers to
```

Comment:
Please note that the folder name needs to be `next-i18next.config.js` due to a filename convention requested by [next-i18next](https://github.com/isaachinman/next-i18next).